TDD_Training/TDD_Training/FizzBuzzService.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System;
+
+namespace TDD_Training
+{
+    public class FizzBuzzService
+    {
+        public string CheckFizzBuzz(int givenNumber)
+        {
+            switch(givenNumber)
+            {
+                case int _ when IsMultipleOfThreeOrFive(givenNumber) == true:
+                {
+                    return "FizzBuzz";
+                }
+
+                case int x when IsMultipleOfThree(givenNumber) == true:
+                {
+                    return "Fizz";
+                }
+
+                case int x when IsMultipleOfFive(givenNumber) == true:
+                {
+                    return "Buzz";
+                }
+                default:
+                {
+                    return Convert.ToString(givenNumber);
+                }
+            }
+        }
+
+        public bool IsMultipleOfThree(int givenNumber)
+        {
```

Comment:
You can do it in single line. the modulus check would by default gives you boolean. you dont have to return it. explicitly.
Can you change that?